### PR TITLE
Added functions to control playlist / volume

### PR DESCRIPTION
**Pull request by shishimaru**
I added 4 util functions to control the playlist and volume.
Could you consider to merge ?

### CODE REVIEW

File: mpv.el [hunk 1]
```diff
@@ -362,6 +383,18 @@ of \\[universal-argument] will add another `mpv-seek-step' seconds."
...
+  "Go to the next entry on the playlist."
+  (interactive)
+  (mpv--enqueue '("playlist-next" "weak") #'ignore))
+
+;;;###autoload
+(defun mpv-playlist-prev ()
+  "Go to the previous entry on the playlist."
+  (interactive)
+  (mpv--enqueue '("playlist-prev" "weak") #'ignore))
+
```
Comment (kljohann) [inline mpv.el:397]: I think "weak" is the default? If this is different for your version of mpv, we can also keep it.
```suggestion
;;;###autoload
(defun mpv-playlist-next ()
  "Go to the next entry on the playlist."
  (interactive)
  (mpv--enqueue '("playlist-next") #'ignore))

;;;###autoload
(defun mpv-playlist-prev ()
  "Go to the previous entry on the playlist."
  (interactive)
  (mpv--enqueue '("playlist-prev") #'ignore))

```

**Review by kljohann — COMMENTED** [review]
Thanks for creating the PR. The volume increase/decrease was already part of #19, but we can add the `playlist-{prev,next}` commands. :+1:

File: mpv.el (hunk 1 above)
Comment (shishimaru) [reply mpv.el:397]: @kljohann 
On Ubuntu20.04LTS, the "weak" is default value.

       playlist-next <flags>
              Go to the next entry on the playlist.

              First argument:

              weak (default)
                     If the last file on the playlist is currently played, do nothing.

              force  Terminate playback if there are no more files on the playlist.

       playlist-prev <flags>
              Go to the previous entry on the playlist.

              First argument:

              weak (default)
                     If the first file on the playlist is currently played, do nothing.

              force  Terminate playback if the first file is being played.

**Comment by shishimaru** [comment]
@kljohann 
I pulled the upstream and resolved the conflict with my forked project with removing volume increase/decrease.
And I removed the default value of playlist-next and playlist-prev commands.
Could you review again ?

**Comment by kljohann** [comment]
Looks good, thanks for your contribution!

**Pull request merged**